### PR TITLE
Cleaned Codes and Removed UnUsed Usings

### PR DESCRIPTION
**Pull request by ghost1372**
i cleaned codes and removed Unused Usings

### CODE REVIEW

File: TLSharp.Core/MTProto/Crypto/AES.cs [hunk 1]
```diff
@@ -216,17 +215,15 @@ public static byte[] XOR(byte[] buffer1, byte[] buffer2)
...
     }
 
-
     // AES engine implementation
 
     public class AesEngine
     {
         // The S box
         private const uint m1 = 0x80808080;
+
```
Comment (knocte) [inline TLSharp.Core/MTProto/Crypto/AES.cs]: why this new line?

File: TLSharp.Core/MTProto/Crypto/AES.cs [hunk 2]
```diff
@@ -576,7 +573,8 @@ public int ProcessBlock(byte[] input, int inOff, byte[] output, int outOff)
             {
                 EncryptBlock(WorkingKey);
             }
-            else {
+            else
+            {
```
Comment (knocte) [inline TLSharp.Core/MTProto/Crypto/AES.cs]: why this change?

File: TLSharp.Core/Auth/Step3_CompleteDHExchange.cs [hunk 3]
```diff
@@ -206,4 +203,4 @@ public Step3_Response FromBytes(byte[] response)
             }
         }
     }
-}
+}
```
Comment (knocte) [inline TLSharp.Core/Auth/Step3_CompleteDHExchange.cs]: why are you changing all the ending of files?

Comment (ghost1372) [reply TLSharp.Core/Auth/Step3_CompleteDHExchange.cs]: i use CodeMaid Cleaner for cleaning code so this is just for increase the readability of the code

Comment (aarani) [reply TLSharp.Core/Auth/Step3_CompleteDHExchange.cs]: i think there is a newline difference here 🤔🤔

Comment (ghost1372) [reply TLSharp.Core/Auth/Step3_CompleteDHExchange.cs]: no new  code, just some usings removed (that dont used) and white spaces removed.

**Comment by aarani** [comment]
Anyway this pull request cannot get merged as entire TeleSharp.TL is generated so every time I generate new layer they will replace and create not necessary git diff

**Comment by knocte** [comment]
@ghost1372 can you separate this PR in two different commits: one that removes unused usings, and one that cleans up the code (but that doesn't touch autogenerated files, like @aarani says)?

**Comment by knocte** [comment]
There are merge conflicts now. If you can rebase and address the feedback requested, please reopen the PR, thanks.
